porting/debug-build/wifi.rst

```diff
@@ -13,6 +13,25 @@ Wifi is fairly easy to get going on most Qualcomm devices. The following should
    echo 1 > /dev/wcnss_wlan
    echo sta > /sys/module/wlan/parameters/fwpath
 
+Errors
+~~~~~~
+
+What to do if the first command line fails with ``/dev/wcnss_wlan: Bad address``
+********************************************************************************
+
+This is not an error condition; don't get put off — try the second command.
+
+What to try if ``/sys/module/wlan`` is missing (not found)
+**********************************************************
+
+Try to load the ``wlan`` kernel module first::
+
+    insmod /system/lib/modules/wlan.ko
+
+If ``insmod`` fails with ``Required key is not found``, you could temporarily disable all the ``CONFIG_MODULE_SIG**`` options in your ``defconfig`` and then re-build ``hybris-boot`` and ``system.img``. **Beware: this disables kernel module signatures (a security feature) — consider fixing it later.**
```

Comment:
You don't need to rebuild system.img when you make changes to your kernel (also probably worth to mention halium-boot)

Reply:
You must rebuild system.img, because it contains kernel modules.

If you fail to do so, you will get `Invalid module format` when loading the module.

Reply:
Oh, that's interesting. Could you please mention that? Specifically that it contains kernel modules. I've never had to rebuild system.img when changing kernel conf, i guess it's worth noting. Apart from that, I approve the PR

Reply:
I made a small change about it, I hope you don't mind

Reply:
Gor, I am absolutely fuming that you would make a small change on my PR >:).

Seriously though, it's fine, though the commit ID (bad2988) confused me when looking through e-mails, because of the word 'bad'.

> I've never had to rebuild system.img when changing kernel conf

This is probably because most kernel config changes don't change the format of kernel modules. But in the case of signed modules, they must have a different format, from what I can tell.

Cheers for merging :)